app/src/main/res/values-bn-rBD/strings.xml

```diff
@@ -505,7 +506,7 @@
   <string name="prescribeddrugsummary_item_name">%1$s %2$s</string>
 
   <!-- Contact patient -->
-  <string name="contactpatient_agreed_to_visit">\&quot;ভিজিটে আসতে সম্মত হয়েছেন&quot;\ বলে রোগীকে চিহ্নিত করুন</string>
```

Comment:
I am curious how the second `&quot;` did not break lint since it is not escaped. 😮

Reply:
I don't think escape character is necessary because we are using `&quot;` instead of `"`?

Reply:
Ah, right. I think the problem was happening when the translations had a apostrophe character.

Anyway, since the processor converts these to escaped quotes and apostrophes, I think it's ok.